test/integration/statefulset/statefulset_test.go

```diff
@@ -397,12 +397,14 @@ func TestStatefulSetStatusWithPodFail(t *testing.T) {
 func TestAutodeleteOwnerRefs(t *testing.T) {
 	tests := []struct {
 		name              string
+		namespace         string
```

Comment:
To reduce toil and repetitive boilerplate we could probably drop the "name" field in the struct and just use the namespace for documentation?

Reply:
Done.